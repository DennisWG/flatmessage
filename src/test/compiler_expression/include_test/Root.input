module Test.Root;

import Test.Include;

message Root
{
	repeated SomeData data;
}