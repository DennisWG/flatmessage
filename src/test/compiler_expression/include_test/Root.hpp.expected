#include "Include.hpp"

namespace Test
{
#pragma pack(push, 1)

    class Root
    {
    public:
        std::vector<SomeData> data;
    };

#pragma pop
}